packages/mobile_app/lib/components/prophecy.dart

```diff
@@ -6,65 +6,92 @@ import 'package:prophecy/bloc.dart';
 import 'package:prophecy_model/prophecy_model.dart';
 import 'prophecy_record.dart';
 import 'prophecy_is_loading.dart';
-import 'package:app/screens/loading.dart';
 import 'package:flutter_svg/flutter_svg.dart';
 import 'package:mutable_wrappers/mutable_wrappers.dart';
 import 'package:users_repository/users_repository.dart';
 import 'package:language/language.dart';
 
 import 'package:int_datetime/int_datetime.dart';
-import 'package:algorithm/astro.dart' show BasicAstrology;
+import 'package:algorithm/algorithm.dart';
 
 class Prophecy extends StatelessWidget {
+  final int dt;
   final UserEntity user;
   final labelStr;
   final birthRow = MutableObject(null);
-  Prophecy({@required this.user})
+  Prophecy({@required this.user, @required this.dt})
       : labelStr = "${user.model.name.capitalize()} (${userRole(user.role)})" {
-    final dt = user.model.birth.toDateTime;
+    //
+    final dtConverted = user.model.birth.toDateTime;
     final sign = user.model.birth.astroSign;
-    final mainPlanet = user.model.birth.astroHousePlanet;
+    //
     birthRow.wrapped = Row(
       children: <Widget>[
         SvgPicture.asset("assets/icons/$sign.svg"),
-        Text(" ${dt.day}.${dt.month}.${dt.year} ",
+        Text(" ${dtConverted.day}.${dtConverted.month}.${dtConverted.year} ",
             style: TextStyle(fontSize: 14)),
-        SvgPicture.asset("assets/icons/$mainPlanet.svg"),
       ],
     );
   }
 
   @override
   Widget build(BuildContext context) {
+    final sign = user.model.birth.astroSign;
+    final userPatron = user.model.birth.astroHousePlanet;
+    final currentPlanets = planetFor[dt.astroSign][sign];
+
     ProphecyBloc prophecyBloc = context.bloc<ProphecyBloc>();
 
     return Padding(
       padding: const EdgeInsets.symmetric(horizontal: 16.0),
       child: BlocBuilder<ProphecyBloc, ProphecyState>(
           bloc: prophecyBloc,
           builder: (context, state) {
+            //
             if (state is ProphecyInitial) {
-              return ProphecyIsLoading(user: this.user);
+              return prophecyIsLoading(
+                  user: this.user, birthRow: birthRow.wrapped);
+              //
+
             } else if (state is ProphecyWasAsked ||
                 state is ProphecyWasClarified) {
+              //
+              /// if big numbers in all prophecies show "impact"
+              /// of user's Patron planet
+              final sum = state.propheciesSum;
+              if (sum > 40.0) currentPlanets[true] = userPatron;
```

Comment:
Would be nice to extract 40.0 to a constant, not clear what it is. (or clarify it in a comment)